search/includes/classes/class-versioning.php

```diff
@@ -227,36 +227,22 @@ public function get_versions( Indexable $indexable, bool $provide_default = true
 		$versions = [];
 
 		if ( $indexable->global ) {
-			$versions = get_site_option( self::INDEX_VERSIONS_OPTION_GLOBAL, array() );
+			$versions = get_site_option( self::INDEX_VERSIONS_OPTION_GLOBAL );
 		} else {
-			$versions = get_option( self::INDEX_VERSIONS_OPTION, array() );
+			$versions = get_option( self::INDEX_VERSIONS_OPTION );
 		}
 
 		$slug = $indexable->slug;
 
-		if ( ! $this->versions_array_has_slug( $versions, $slug ) ) {
-			if ( $provide_default ) {
-				return array(
-					1 => array(
-						'number'         => 1,
-						'active'         => true,
-						'created_time'   => null, // We don't know when it was actually created
-						'activated_time' => null,
-					),
-				);
-			} else {
-				return [];
-			}
+		if ( ! is_array( $versions ) || ! isset( $versions[ $indexable->slug ] ) ||
```

Comment:
could we just use. the $provide_default code? We can change the default value to false there.